File: PHP Word Examples/ReplaceTextApiDemo/replaceWithText.php
<?php

use Spire\Cloud\Word\Sdk\Client\Configuration;
use Spire\Cloud\Word\Sdk\Client\Api\ReplaceTextApi;

$appId="your id";
$appKey="your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId,$appKey,$baseUrl);
$apiInstance = new ReplaceTextApi($configuration);

$fileName = "ReplaceText.docx";
$old_value = '#test';
$new_value = "E-iceblue";
$match_case = false;
$match_whole_word = null;
$replace_first = null;
$password = null;
$folder = "input";
$storage = null;
$dest_file_path = "/ReplaceText/ReplaceWithText_false.docx";
$apiInstance->replaceWithText(
    $fileName,
    $old_value,
    $new_value,
    $dest_file_path,
    $password,
    $folder,
    $storage,
    $match_case,
    $match_whole_word,
    $replace_first
);

File: Python Word Examples/ReplaceTextApiDemo/replaceWithText.py
import spirecloudword
from spirecloudword.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudword.api.ReplaceTextApi(configuration)

file_name = 'ReplaceText.docx'
old_value = '#test'
new_value = 'E-iceblue'
extra = {'folder': "input", 'match_case': 'true', 'match_whole_word': 'true', 'replace_first': 'false','dest_file_path': 'output/ReplaceWithText_true_true_false.docx'}
result =api.replace_with_text(file_name, old_value, new_value, **extra)